resolver/ng/init.sls

```diff
@@ -0,0 +1,42 @@
+{% from slspath + "/map.jinja" import resolver with context %}
```

Comment:
In another PR (cannot remember which one) the '**slspath**' was introduced but the reviewer was unsure and deferred to another Salt Developer for advice, and PR remained open.  I am also unsure if 'slspath' is approved by saltstack-formulas so pinging @gtmanfred for opinion.  Otherwise I presume this is okay.

Reply:
It's available officially here, even for older version (2016.11).
https://docs.saltstack.com/en/latest/ref/states/vars.html

Reply:
@noelmcloughlin do you have a link to that other pr so that I can review the reason it wasn't approved back then?

Reply:
It must have been this one I was thinking about:  https://github.com/saltstack-formulas/chrony-formula/pull/8

Reply:
ok, looks like that is waiting for a review from someone else, I have no problem with slspath being used, it has been in for long enough.

Reply:
Sounds good to me too.